assets/css/styles.css

```diff
@@ -547,7 +580,7 @@ p.tracks-des {
 }
 
```

Comment:
Before:
<img width="312" alt="screen shot 2018-11-14 at 9 53 00 pm" src="https://user-images.githubusercontent.com/1021101/48533049-cdf53c80-e857-11e8-9cf6-65a23df66a47.png">


After:
```
.time-box {
    transform: translate(-65px);
}
```

Reply:
it's now flush on all screen sizes